recipe/meta.yaml

```diff
@@ -12,16 +12,15 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -v --no-deps --no-build-isolation
 
```

Comment:
```suggestion
skip: True  # [py<38]  
```

Reply:
Oh wait, you are asking ME to be explicit. Sorry.

Reply:
Done